ansible/roles/test/files/ptftests/arista.py

```diff
@@ -313,6 +331,49 @@ def parse_bgp_route(self, output, expects):
 
         return set(expects) == prefixes
 
+    def get_bgp_neighbors(self):
+        neigh_bgp = {}
+        dut_bgp = {}
+        for cmd, ver in [('show ip bgp neighbors', 'v4'), ('show ipv6 bgp neighbors', 'v6')]:
```

Comment:
>for cmd, ver [](start = 8, length = 12)

you don't need a for-loop such small range.

Reply:
I need to do the same set of operations in both cases, hence the loop.